automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/SignInSpec.scala

```diff
@@ -13,14 +12,17 @@ class SignInSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Cle
     "with a registered account" - {
 
       "should be able to log in and out multiple times as multiple users" in withWebDriver { implicit driver =>
-        withSignIn(Config.Users.draco) { listPage =>
-          listPage.readUserEmail() shouldEqual Config.Users.draco.email
+        val users = UserPool.chooseStudent(2)
+        val user1 = users.head
+        val user2 = users(1)
```

Comment:
I think we should be able to make the assignment to `user1` and `user2` a little cleaner but I couldn't get it to work in the 10 minutes I spent on it. Just mentioning it here but let's save it for later clean-up.